frontend/src/bundles/search-candidates/pages/candidates.tsx

```diff
@@ -162,16 +162,30 @@ const Candidates: React.FC = () => {
                     <Grid
                         className={getValidClassNames(
                             styles.searchResults,
+                            filteredCandidates.length > 0 && styles.scroll,
                             isFilterOpened ? styles.searchResultsHidden : '',
                         )}
                     >
-                        {filteredCandidates.map((candidate) => (
-                            <CandidateProfile
-                                key={candidate.id}
-                                isProfileCard
-                                candidateData={candidate}
-                            />
-                        ))}
+                        {filteredCandidates.length > 0 ? (
+                            filteredCandidates.map((candidate) => (
+                                <CandidateProfile
+                                    key={candidate.id}
+                                    isProfileCard
+                                    candidateData={candidate}
+                                />
+                            ))
+                        ) : (
+                            <Grid container item className={styles.noResults}>
+                                <Typography
+                                    className={styles.noResultsText}
+                                    variant="body1"
+                                >
+                                    {' '}
+                                    No candidates were found, try to change the
+                                    filtering{' '}
```

Comment:
why do we need `{' '}`?

Reply:
fixed. 
strange, I didn't write that. maybe vs code inserted it for some reason